### PR TITLE
Rc 311

### PR DESCRIPTION
**Pull request by prckent**
Includes jastrow cutoff bugfix

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+
+* Added numerical tolerance to check of jastrow cutoff and Wiger Seitz
+  radius.
+* Cmake correctly configures when MPI is not present.
+* Improved support for test coverage measurements.
+* Added unit tests for some estimators.
+
+### NEXUS
+
+* ipython compatibile exit handling (from Duy Le)
```
Comment (markdewing) [inline CHANGELOG.md]: Spelling of compatible and capitilization of IPython.
"IPython-compatible exit handling (from Duy Le)"

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
 
+## [3.1.1] - 2017-08-01
+
+### Notes
+
+This is a bugfix release and recommended update.
+
+### QMCPACK updates
+
+* Added numerical tolerance to check of jastrow cutoff and Wiger Seitz
```
Comment (markdewing) [inline CHANGELOG.md]: Wigner-Seitz

Comment (prckent) [inline CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (prckent) [inline CHANGELOG.md]: Fixed

**Comment by ye-luo** [comment]
how about issue #290 ? We list it as a known issue or do a quick fix?

**Comment by prckent** [comment]
Re: #290 , since this was not on the milestone and has not been fixed yet, no.

**Comment by prckent** [comment]
Note that rhea at OLCF is down today for some reason. Lets see if the CI resumes working when rhea returns.

**Comment by ye-luo** [comment]
I marked issue #290 for 3.2 release.

**Comment by grahamlopez** [comment]
OLCF is having a site-wide planned outage today. I'll watch for things to come back and make sure that CI runs

**Comment by grahamlopez** [comment]
test this please (to ensure latest commits included)

**Comment by grahamlopez** [comment]
Note that even though Rhea reports as back up, Jenkins hasn't been able to submit a job, and I haven't been able to log in through ssh yet.

**Comment by grahamlopez** [comment]
test this please

**Pull request merged**